static/js/components/Dashboard/Dashboard.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from "react";
 import { Strip } from "@canonical/react-components";
-import { InterviewParticipation } from "./InterviewParticipation";
-import { JobsDetails } from "./JobsDetails";
-import { Workload } from "./Workload";
+import InterviewParticipation from "../InterviewParticipation";
+import JobsDetails from "../JobsDetails";
+import Workload from "../Workload";
 
 export const Dashboard: React.FC<any> = () => {
```

Comment:
You don't need to type components with `React.FC` any longer. This can simply be the following if you don't have any props defined.

```suggestion
export const Dashboard = () => {
```

See https://github.com/facebook/create-react-app/pull/8177 for more information

Reply:
:+1: